api/helpers.go

```diff
@@ -27,7 +27,9 @@ func httpWriteJSON(w http.ResponseWriter, data interface{}) {
 		log.Warnw("failed to write on response", "error", err)
 		return
 	}
-	log.Debugw("api response", "bytes", n, "data", strings.ReplaceAll(string(jdata), "\"", ""))
+	if !DisabledLogging && log.Level() == log.LogLevelDebug {
```

Comment:
if the log is initialized with the correct level, the second condition does not take any effect no? If the log is initialized with `Info` level, any `DebugX` should not print anything...